index.html

```diff
@@ -413,6 +416,30 @@ <h3>
       </li>
     </ul>
   </p>
+  <p>
+    The <dfn>change in contributing factors is substantial</dfn> steps are as follows:
+    <ol>
+      <li>
+        If [=implementation-defined=] low-level hardware metrics that contribute to the
+        system performance and [=current pressure state=] drop below or exceed an
+        [=implementation-defined=] threshold for the [=current pressure state=], return true.
+      </li>
+      <li>
+        Return false.
+      </li>
+    </ol>
+    <aside class="note">
+      The [=change in contributing factors is substantial=] algorithm allows user agents to avoid
+      flip-flopping between two states in certain circumstances. For example, a state might otherwise
+      change too rapidly in response to a certain system metric that fluctuates around a boundary
+      condition that triggers a state change. This specification does not define
+      the precise algorithm to allow implementations optimize this algorithm to meet their specific needs.
```

Comment:
Maybe not just "meet their specific needs" as it sounds a bit like impls will have very different behavior. Maybe 'to better match the behavior of the underlying hardware" ? or something similar

Reply:
Addressed in an update to this PR.